paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -168,6 +170,19 @@ internal class DefaultFlowController @Inject internal constructor(
         configuration: PaymentSheet.Configuration?,
         callback: PaymentSheet.FlowController.ConfigCallback
     ) {
+        try {
+            configuration?.validate()
+            if (clientSecret.value.isBlank()) {
+                throw InvalidParameterException("Client secret cannot be blank.")
```

Comment:
can we be more explicit in our messaging here, like "PaymentIntent client_secret cannot be an empty string"

Reply:
and same for any other places we mention "blank", I'm not sure it is explicit enough